docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+  web:
+    build: .
+    command: uvicorn app.main:app --host 0.0.0.0 --reload
+    container_name: wira_app
+    volumes:
+      - ./app:/code/app
+      - ./tests:/code/tests
+      - ./data:/code/app/data
```

Comment:
all data is volume mounted, so images scraped and 'uploaded' will end up in the repository `data/` directory.